### PR TITLE
Restructured the project into folders, made use of configuration object

### PR DESCRIPTION
**Pull request by almenjonatan**
Read commit message, do not merge without trying out and  feedback =)

### CODE REVIEW

File: src/csv_reader.py [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+    @staticmethod
+    def get_data(datatype):
+        data, label = CsvReader.readfile(datatype)
+        processed = [CsvReader._process(text) for text in data]
+        return processed, label
+
+    @staticmethod
+    def _process(text):
+        # replaces each number with NUMTOKEN
+        text_with_token = re.sub('\s+NUMTOEKN\s+', ' NUMTOKEN ',
```
Comment (Oliv95) [inline src/csv_reader.py]: Fel i regex, NUMTOEKN

File: src/csv_reader.py [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                                 re.sub('\d+', 'NUMTOKEN', text))
```
Comment (Oliv95) [inline src/csv_reader.py]: borde va ' NUMTOKEN '

**Comment by Mxiim** [comment]
Remove panda dependency.
Fix write objects.
Attritube for name in config file for a network. 
Refactor the constant "zero" when reading yamlconfig. 
Logger object in code /model.

**Comment by Mxiim** [comment]
Most of the major problems are fixed, if not all. Feedback is welcome.

File: src/helper.py [hunk 3]
```diff
@@ -80,23 +81,10 @@ def get_indicies(sentence, dictionary, max_words):
     return result
 
 
-def pad(sentence, dictionary, max_words):
```
Comment (Jaxing) [inline src/helper.py]: Why did you remove padding?

File: src/csv_reader.py [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+# ==============================================================================
+from enum import Enum
+
+import re
+from definitions import DATASETS_PATH
+import pandas as pd
```
Comment (Jaxing) [inline src/csv_reader.py]: pandas is unused.

File: src/helper.py (hunk 3 above)
Comment (Mxiim) [reply src/helper.py]: Mistake, didn't take the updated csvreader.

File: src/folder_builder.py [hunk 5]
```diff
@@ -19,12 +19,15 @@ def build_structure(config):
     dir_to_create = LOGS_DIR + '/' + type + '/' + name
     tensor_dir_train_to_create = dir_to_create + '/' + TENSOR_DIR_TRAIN
     tensor_dir_valid_to_create = dir_to_create + '/' + TENSOR_DIR_VALID
-    print(tensor_dir_train_to_create)
-    print(tensor_dir_valid_to_create)
+    checkpoings_dir_to_create = dir_to_create + '/' + CHECKPOINTS_DIR
```
Comment (Jaxing) [inline src/folder_builder.py]: spelling error checkpoings
